js/ui/control/fullscreen.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Control = require('./control');
+var DOM = require('../../util/dom');
+var util = require('../../util/util');
+var window = require('../../util/window');
+
+module.exports = Fullscreen;
+
+function Fullscreen(options) {
+    util.setOptions(this, options);
+}
+
+Fullscreen.prototype = util.inherit(Control, {
+    options: {
+        position: 'top-left'
+    },
+
+    onAdd: function(map) {
+        var className = 'mapboxgl-ctrl';
+        var container = this._container = DOM.create('div', className + '-group', map.getContainer());
+        var button = this._fullscreenButton = DOM.create('button', (className + '-icon ' + className + '-fullscreen'), this._container);
+        button.id = 'fullscreen-button';
```

Comment:
Avoid assigning an ID -- IDs are required to be unique, and there may be more than one map with fullscreen control on a page.